bus/client.go

```diff
@@ -491,6 +494,32 @@ func (c *Client) GougingParams() (gp api.GougingParams, err error) {
 	return
 }
 
+// Accounts returns the ephemeral accounts for a given owner.
+func (c *Client) Accounts(owner string) (accounts []ephemeralaccounts.Account, err error) {
+	err = c.c.GET(fmt.Sprintf("/accounts/%s", api.ParamString(owner)), &accounts)
```

Comment:
why can't the owner be a simple string?

Reply:
Because jape error. The types need to implement `TextMarshaler` which `string` doesn't do as a native type.

Reply:
can you add a comment to `ParamString` perhaps, feels a bit silly we have to work around `jape` this way